tests/integration_tests/test_sklearn.py

```diff
@@ -308,6 +310,7 @@ def test_objective_error_score_invalid() -> None:
 
 # TODO(himkt): Remove this method with the deletion of deprecated distributions.
 # https://github.com/optuna/optuna/issues/2941
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
In this test function, old distributions such as `UniformDistribution` and `LogUniformDistribution`, and supressing `FututreWarning` sounds reasonable.